csharp/src/SeedLang.Block/BxfObject.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Newtonsoft.Json;
```

Comment:
I roughly scanned the comparison doc

https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to?pivots=dotnet-5-0

and thought that it will not be a big deal to switch the API from any one of them to the other.

I chose Newtonsoft Json just because of a single and trivial reason: .NET5 has built-in System.Text.Json support but .netstandard 2.0 requires installing an external package. Just hesitated a bit about how to make our project file consistent for this case... Then without doing enough experiments, I decided to use Newtonsoft Json for now to bypass this trivial issue.